ynr/apps/official_documents/views.py

```diff
@@ -48,6 +48,8 @@ def form_valid(self, form):
         """
         self.object = form.save()
         try:
+            if hasattr(self.object.ballot, "rawpeople"):
+                self.object.ballot.rawpeople.all().delete()
```

Comment:
The `rawpeople` relation is a OneToOne so called `all()` will raise an error so this should be `self.object.ballot.rawpeople.delete()`

Also I can't quite remember the context of this change, but perhaps we might only want to delete the raw people if the SOPN was parsed successfully? As when it fails the raw people might be useful? If so we could move this to just above the `except` line